src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -296,27 +293,13 @@ void GcsPlacementGroupScheduler::CommitAllBundles(
                                       node_id,
                                       schedule_failure_handler,
                                       schedule_success_handler](const Status &status) {
+      auto commited_bundle_locations = std::make_shared<BundleLocations>();
       for (const auto &bundle : bundles_per_node) {
         lease_status_tracker->MarkCommitRequestReturned(node_id, bundle, status);
-
-        auto resources = bundle->GetFormattedResources();
-        // For gcs actor scheduler, placement group's wildcard resources have to be
-        // updated incrementally.
-        // TODO(Chong-Li): This part should be removed when PG scheduling is refactored.
-        if (RayConfig::instance().gcs_actor_scheduling_enabled()) {
-          auto wildcard_resources = bundle->GetWildcardResources();
```

Comment:
NOTE: The accumulates of wildcard resources will be done in the following  `CommitBundleResources` method.